token_reset/jupyterlab-github.txt

```diff
@@ -1,2 +1,3 @@
 # token reset failed for these packages - trying again later
+# --without-travis
```

Comment:
I've never seen these magic comments. Do we have them documented somewhere?

Reply:
No that is my bad. We have so little documentation of anything really.

Reply:
Maybe we can add to [the README section on token resets]( https://github.com/conda-forge/admin-requests#reset-your-feedstock-token )?